client/packages/lowcoder/src/i18n/locales/en.ts

```diff
@@ -2111,8 +2111,8 @@ export const en = {
   },
   queryTutorial: {
     js: "",
-    transformer: "https://docs.lowcoder.dev/build-apps/write-javascript/transformers",
-    tempState: "https://docs.lowcoder.dev/build-apps/write-javascript/temporary-state",
+    transformer: "https://docs.lowcoder.cloud/build-apps/write-javascript/transformers",
```

Comment:
https://docs.lowcoder.cloud/business-logic-in-apps/write-javascript/transformers